crates/iota/src/name_commands.rs

```diff
@@ -58,6 +58,11 @@ use crate::{
 /// The overbid must be at least of 1 IOTA, which is 10^9 NANOs
 const MIN_OVERBID: u64 = 1_000_000_000;
 
+/// Minimum coin amount (in NANOs) required for gas payment eligibility.
+/// Coins below this value (9_000_000 NANOs = 0.009 IOTA) are ignored for gas
+/// payment.
+const MIN_COIN_AMOUNT_FOR_GAS_PAYMENT: u64 = 9_000_000;
```

Comment:
Is this something we may have to change in the future because gas config changes?